Description: Fix Python libdir not found on Debian et al.
Author: Martin Wimpress <code@flexion.org>
 
Index: python-caja/configure.ac
===================================================================
--- python-caja.orig/configure.ac
+++ python-caja/configure.ac
@@ -40,7 +40,7 @@ dnl * Check for Python
 dnl **************************************************
 AM_PATH_PYTHON([2.7])
 PKG_CHECK_MODULES([PYTHON], [python-${PYTHON_VERSION}])
-PYTHON_LIB_LOC="`pkg-config python-${PYTHON_VERSION} --variable=libdir`"
+PYTHON_LIB_LOC="`pkg-config gobject-introspection-1.0 --variable=libdir`"
 PYTHON_ABIFLAGS=`$PYTHON -c 'import sys; exec("try: print (sys.abiflags)\nexcept: pass")'`
 AC_SUBST(PYTHON_LIBS)
 AC_SUBST(PYTHON_CFLAGS)
